protocol/git/README.md

```diff
@@ -106,6 +106,7 @@ View a list of new commits. View changed files. Merge branch into master.
     git log origin/master..<branch-name>
     git diff --stat origin/master
     git checkout master
+    git pull
```

Comment:
If your local `master` is not up to date, you won't be allowed to push to master. So I think this is implicitly solved.

I'd be fine merging this PR in with a change to `pull --rebase` instead of `pull`